### DISCUSSION TITLE
How to make ojs code self contained in html or revealjs output

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#6165] [Q&A]
### Description

Hello, 
I like to make Revealjs presentations files that I can share so it is important for those files to be self contained. 

This issue discusses that the self-contained option does not work when using ojs: https://github.com/quarto-dev/quarto-cli/issues/1412

Is there any workarounds? One is mentioned in the issue but it is not clear to me how to implement it.

**Comment by cscheid** [comment]
No, that's not something we can control. The observableJS runtime has a number of places where it makes network calls to get its dependencies, and it would not be practical for us to fix all of them in a timely manner.

**Comment by mcanouil** [comment]
To note the same answer was provided in the opened issue you (@victorfeagins) commented on earlier thus closing this in favour of the opened issue:

- #1412

Regarding the "workaround", it's not really a workaround as it requires a pretty fair amount of work which is not trivial.
